### ISSUE TITLE
Replacing plstore prompt with password-store

### ISSUE THREAD

**Issue by d-miketa** [kidd/org-gcal.el#225]
Hi and thanks for this plugin!

I was wondering if it's possible to integrate the plstore mechanism with a password manager. I already use `password-store` for my Calendar secret and have to input a password once already to recover it for `org-gcal`. It's annoying to have to do it again for another prompt immediately following this one. A better behaviour would be to specify a function outputting the plstore password. It'd also be great to be able to specify a function for the org-gcal secret.

**Comment by podiki** [comment]
Yes, can't we plug into the options emacs has for storing secrets (e.g. authinfo, gpg)? I was very confused by the plstore pop up and still trying to figure out why it doesn't actually save...

**Comment by megodoonch** [comment]
I don't know if this is related, but I have either a new or newly-noticed error on sync: `org-gcal-sync-buffer: error: (no-catch block nil)`

Here is my org-gcal related .emacs bit:

```
(setq org-gcal-client-id "[redacted]apps.googleusercontent.com"
       org-gcal-client-secret "[redacted]"
       org-gcal-fetch-file-alist '(("[redacted]" .  "~/Sync/google.org") 
				   ("[redacted]@import.calendar.google.com" . "~/Sync/uu.org")
				   ([redacted]@group.calendar.google.com" . "~/Sync/tasks.org")
                                   ))

(require 'org-gcal)
(setq epg-pinentry-mode 'loopback)
(setenv "GPG_AGENT_INFO")
(setq-default plstore-cache-passphrase-for-symmetric-encryption t)
```

**Comment by telotortium** [comment]
I recently discovered that `plstore` can use asymmetric GPG keys from `gpg-agent` and wrote some instructions on how to use it. It at least doesn't prompt me for passwords so much. Try setting this up and see if it works? https://github.com/kidd/org-gcal.el/pull/239

**Comment by eval-on-point** [comment]
I am attempting this and see a small error in the README:

```
    (require 'plstore)
    (add-to-list 'plstore-encrypt-to '("KEY ID"))
```
produces an error because `plstore-encrypt-to` expects to be a list of strings
